<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>536</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>CLIP</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,999">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,999,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btn_select_video">
            <property name="text">
             <string>SelectVideo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bn_stop">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="rowCount">
           <number>3</number>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>3</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Text</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Score</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>normal</string>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string>0.0</string>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>violence</string>
           </property>
          </item>
          <item row="1" column="1">
           <property name="text">
            <string>0.0</string>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>fight</string>
           </property>
          </item>
          <item row="2" column="1">
           <property name="text">
            <string>0.0</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btn_add_text">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_remove_text">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="myQLabel" name="label">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQLabel</class>
   <extends>QLabel</extends>
   <header>myqlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
